snuba/datasets/__init__.py

```diff
@@ -62,3 +62,10 @@ def column_expr(self, column_name, body):
 
         """
         return column_name
+
+    def get_bulk_loader(self, source, dest_table):
```

Comment:
Passing the source in input allows us to abstract the source away from the process to insert data into clickhouse. It also allows us to reuse the same instance if we decide to load multiple datasets at the same time.

Reply:
Makes sense